### PR TITLE
fix: graph content pushed down when the text is too long

### PR DESCRIPTION
**Pull request by MounirDhahri**
### Description

This PR resolves [CX-2812]

When the content is too tall, the screen goes down. this won't happen after this fix since the text has fixed height now

**Before**

https://user-images.githubusercontent.com/11945712/186169349-3e55f7d1-d244-493c-8c16-9532a7aae509.mov

**After** Even if the text is long, the graph will keep position
Group 4

#### Acceptance Criteria

-

#### Setup Instructions

-

### PR Checklist

- [x] I tested my changes on **iOS** / **Android**.
- [x] I added screenshots or videos to illustrate my changes.
- [ ] I added Tests and Stories for my changes.
- [ ] I added an [app state migration].
- [ ] I hid my changes behind a [feature flag].
- [ ] I have prefixed changes that need to be tested during a release QA with **[NEEDS EXTERNAL QA]** on the changelog.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- fix graph content pushed down when the text is too long - mounir

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[CX-2812]: https://artsyproduct.atlassian.net/browse/CX-2812?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by olerichter00 — DISMISSED** [review]
Thanks for the fix! 🌟 

Have you tested the fix with font scaling? A fixed height might be not enough and we might want to multiply it by the scaling factor.

**Comment by MounirDhahri** [comment]
@olerichter00 I haven't honestly, but If we expect to support font scaling there, I think we will also need to adjust other parts of the screen and I am leaning towards doing this separately

**Comment by olerichter00** [comment]
Either way works for me. We can do it now or create a separate ticket.

Personally, I think that with any feature, we should make sure that the design supports font scaling to make the app accessible for all users.

**Comment by MounirDhahri** [comment]
Good idea, I created a ticket for it and added it to the CX Up nex https://artsyproduct.atlassian.net/browse/CX-2825

**Comment by olerichter00** [comment]
The rest of the screen looks ok with font scaling. Just the bottom margin is missing ([Ticket](https://artsyproduct.atlassian.net/browse/CX-2824)).

File: src/palette/elements/LineGraph/LineGraphHeader.tsx [hunk 1]
```diff
@@ -28,9 +28,13 @@ export const LineGraphHeader: React.FC<LineGraphHeaderProps> = ({
         </Flex>
       )}
       {!!text && (
-        <Text variant="xs" color="black60">
-          {text}
-        </Text>
+        // We need to have a fixed height here to make sure in case the text is too long,
+        // it doesn't push the graph content down
+        <Flex height={40}>
```
Comment (olerichter00) [inline src/palette/elements/LineGraph/LineGraphHeader.tsx:33]:
```suggestion
        <Flex height={PixelRatio.getFontScale() * 40}>
```

**Review by olerichter00 — COMMENTED** [review]
Well, let's fix it now. Picking up the ticket later will be more work.

**Pull request merged**